External/FEXCore/Source/Interface/Core/ObjectCache/NamedRegionObjectHandler.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Interface/Context/Context.h"
+#include "Interface/Core/ObjectCache/ObjectCacheService.h"
+
+#include <FEXCore/Config/Config.h>
+
+namespace FEXCore::CodeSerialize {
+  NamedRegionObjectHandler::NamedRegionObjectHandler(FEXCore::Context::Context *ctx) {
+    DefaultSerializationConfig.Cookie = CODE_COOKIE;
+
+    // Initialize the Arch from CPUID
+    uint32_t Arch = ctx->CPUID.RunFunction(0x4000'0001, 0).eax & 0xF;
+    DefaultSerializationConfig.Arch = Arch;
+
+    DefaultSerializationConfig.MaxInstPerBlock = ctx->Config.MaxInstPerBlock;
```

Comment:
`MaxInstPerBlock` and `Multiblock` also need to be included here?

Reply:
Mixing them up should be fine, though we may or may not want that

Reply:
Only one missing here is Multiblock since Multiblock isn't supported with the code cache yet.

Reply:
Mutliblock is actually included in this PR, though neither `MaxInstPerBlock` nor `Multiblock` should change our ABI.

`HostFeatures` makes sense.